Odysee/Controllers/Content/CommentsViewController.swift

```diff
@@ -503,8 +513,8 @@ class CommentsViewController: UIViewController, UITableViewDelegate, UITableView
             return
         }
 
-        if channels.count == 0 {
-            showMessage(message: String.localized("You need to create a channel before you can react to comments"))
+        if currentCommentAsIndex == -1 || channels.count == 0 {
+            showMessage(message: String.localized("You need to select a channel to react to comments"))
```

Comment:
We need to show a different message if `channels.count == 0` which lets the user know that they should create a channel.

Reply:
Done